package.json

```diff
@@ -64,7 +64,6 @@
 		"istanbul-middleware": "=0.2.2",
 		"js-yaml": "=3.10.0",
 		"json-refs": "=3.0.3",
-		"json-schema": "=0.2.3",
```

Comment:
@diego-G I am not sure, if that really is  a `unused library` Because I am sure `z-schema` can't work without `json-schema`.